Chrome docs (#11747)
fleetdm/fleet: docs/Using-Fleet/ChromeOS.md

# ChromeOS

## Adding ChromeOS hosts to Fleet
Fleet provides a Chrome extension which you can install via Google Admin.

To learn how to add ChromeOS hosts to Fleet, visit [here](https://fleetdm.com/docs/using-fleet/adding-hosts#add-chromebooks-with-the-fleetd-chrome-extension).

## Available tables
To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables).

## Setting the hostname
By default, the hostname for a Chromebook host will be blank. The hostname can be customized in Google Admin by configuring the ["device network hostname template"](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information).

## Current Limitations in ChromeOS

> - Scheduled queries are currently not available in ChromeOS

Author: ```suggestion
- Scheduled queries are currently not available in ChromeOS
- More tables will be added in https://github.com/fleetdm/fleet/issues/11037
```

# ChromeOS

## Adding ChromeOS hosts to Fleet
Fleet provides a Chrome extension which you can install via Google Admin.

To learn how to add ChromeOS hosts to Fleet, visit [here](https://fleetdm.com/docs/using-fleet/adding-hosts#add-chromebooks-with-the-fleetd-chrome-extension).

## Available tables

> To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables).

Reviewer: Looks good, suggestion for future iteration: maybe @eashaw can build the tables to keep the platform type in a url param so we can directly link to chromeos tables only (e.g., `https://fleetdm.com/tables/account_policy_data&platform=chromeos`) 

@zhumo I'll let you file and prioritize if you agree

What that would look like:
<img width="1442" alt="Screenshot 2023-06-09 at 9 24 15 AM" src="https://github.com/fleetdm/fleet/assets/71795832/ab3adf90-a64e-401d-ab1f-b24a6cd76202">

Vs. what we currently see clicking this link
<img width="1435" alt="Screenshot 2023-06-09 at 9 24 52 AM" src="https://github.com/fleetdm/fleet/assets/71795832/37c085ac-1fdd-4439-875f-71d892371693">

Reviewer: @zhumo actually, this link works better since it has the platform filtered to chrome: https://fleetdm.com/tables/chrome_extensions?platformFilter=chrome

Can we swap https://fleetdm.com/tables?

Author: oh thanks! I was looking for this. When I click the platform filter, the URL params don't change, so I assumed that doesn't exist! will change.

Author: ```suggestion
To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables/chrome_extensions?platformFilter=chrome).
```

## Setting the hostname

> By default, the hostname for a Chromebook host will be blank. The hostname can be customized in Google Admin by configuring the ["device network hostname template"](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information).

Reviewer: If we want to specify the drill down, we can add to that second sentence. I had to use `cmd + f` to find "device network hostname template" in both the settings and the docs since it wasn't clear.

"The hostname can be customized in Google Admin by  Under Devices > Chrome > Settings > Device > Device Settings > Other Settings, configure ["Device network hostname template"](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information)."

Screenshot of drill down:
<img width="1477" alt="Screenshot 2023-06-09 at 9 34 24 AM" src="https://github.com/fleetdm/fleet/assets/71795832/a955fa38-6845-4058-8da1-9feb92b09525">

Author: ```suggestion
By default, the hostname for a Chromebook host will be blank. The hostname can be customized in Google Admin under Devices > Chrome > Settings > Device > Device Settings > Other Settings > [Device network hostname template](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information).
```
